src/components/Firestore/FirestoreEmulatedApiProvider.tsx

```diff
@@ -35,17 +48,18 @@ const FIRESTORE_OPTIONS = {};
  * the Emulator Hub.
  */
 export const FirestoreEmulatedApiProvider: React.FC<
-  React.PropsWithChildren<{
-    disableDevTools?: boolean;
-  }>
-> = React.memo(({ children, disableDevTools }) => {
+  React.PropsWithChildren<unknown>
```

Comment:
Is there a reason not to do this?

```suggestion
  React.PropsWithChildren<{}>
```